packages/terra-time-input/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss

```
@@ -55,7 +55,7 @@
 
   --terra-time-input-disabled-background-color: #222a2e;
   --terra-time-input-disabled-border: 1px solid #181b1d;
```

Super nit -- looks like these color values were also incorrect and now should be updated to match [/terra-form-input/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss#L8-L9](https://github.com/cerner/terra-core/blob/784426174ed71c67dc1f97df6bd464b877b50029/packages/terra-form-input/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss#L8-L9)
```suggestion
  --terra-time-input-disabled-background-color: #1d2529;
  --terra-time-input-disabled-border: 1px solid #242b2b;
```